packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -102,10 +115,11 @@ class IrisGridTableModel
     (
       layoutHintsFrozenColumns?: ColumnName[],
       userFrozenColumns?: ColumnName[]
-    ): ColumnName[] => userFrozenColumns ?? layoutHintsFrozenColumns ?? []
+    ): ColumnName[] | readonly ColumnName[] =>
```

Comment:
Input parameters should also be `readonly ColumnName[]`
```suggestion
    ): readonly ColumnName[] =>
```